flink-sql-connector-tdsql-cdc/pom.xml

```diff
@@ -0,0 +1,84 @@
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>flink-cdc-connectors</artifactId>
+        <groupId>com.ververica</groupId>
+        <version>2.3-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>flink-sql-connector-tdsql-cdc</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-connector-tdsql-cdc</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.4</version>
+                <executions>
+                    <execution>
+                        <id>shade-flink</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <shadeTestJar>false</shadeTestJar>
+                            <artifactSet>
+                                <includes>
+                                    <include>com.ververica:flink-connector-debezium</include>
+                                    <include>com.ververica:flink-connector-tdsql-cdc</include>
+                                    <include>org.tikv:tikv-client-java</include>
+                                    <include>com.google.protobuf:*</include>
+                                    <include>io.grpc:*</include>
+                                    <!--  Include fixed version 18.0-13.0 of flink shaded guava  -->
+                                    <include>org.apache.flink:flink-shaded-guava</include>
+                                </includes>
+                            </artifactSet>
+                            <relocations>
+                                <relocation>
+                                    <pattern>com.google</pattern>
+                                    <shadedPattern>
+                                        com.ververica.cdc.connectors.shaded.com.google
+                                    </shadedPattern>
+                                </relocation>
+                                <relocation>
+                                    <pattern>io.grpc</pattern>
+                                    <shadedPattern>
+                                        com.ververica.cdc.connectors.shaded.io.grpc
```

Comment:
Hi, here include and relocation is not right. Maybe, you should refer to pom of flink-sql-connector-mysql-cdc.

Reply:
@gong Thanks for point out this problem. I fixed it now. Did you find other problem? by the way, can you help me do code review? thanks a lot!

Reply:
> @gong Thanks for point out this problem. I fixed it now. Did you find other problem? by the way, can you help me do code review? thanks a lot!

@guixin I encounter network problems when I use this connector to test TDSQL districbute database. So I  can't conntiune to verify debezium exception problem. I think that debezium exception should need modify debezium to solve problem.

Reply:
> @gong network problems? Is flink runtime can't connect to tdsql set in cluster? use `/*proxy*/show status;` command to find tdsql cluster set info and try to connect one. example from TDSQL product wiki: https://cloud.tencent.com/document/product/557/8768
> 
> ```shell
> mysql> /*proxy*/show status;
>     +-----------------------------+------------------------------------------------------------------------------+
>     | status_name                 | value                                                                        |
>     +-----------------------------+------------------------------------------------------------------------------+
>     | cluster                     | group_1499858910_79548                                                       |
>     | set_1499859173_1:ip         | 10.49.118.165:5025;10.175.98.109:5025@1@IDC_4@0,10.231.23.241:5025@1@IDC_2@0 |
>     | set_1499859173_1:hash_range | 0---31                                                                       |
>     | set_1499911640_3:ip         | 10.49.118.165:5026;10.175.98.109:5026@1@IDC_4@0,10.231.23.241:5026@1@IDC_2@0 |
>     | set_1499911640_3:hash_range | 32---63                                                                      |
>     | set                         | set_1499859173_1,set_1499911640_3                                            |
> ```
> 
> try to connect set_1499859173_1: `mysql -h10.49.118.165 -P5025 -uxxx -pxxx`

I have network problem when I use public address to test.